Newsletters: add 261 (2023-07-26)

harding opened the pull request:
- [x] Lede, releases/RCs, topic entries @harding 
- [x] Bitcoin SE section @bitschmidty 
- [x] Bitcoin Core GUI 740 @murchandamus

LarryRuane commented on _posts/en/newsletters/2023-07-26-newsletter.md:
```diff
@@ -0,0 +1,203 @@
+      [channel jamming attacks][topic channel jamming attacks].  The
+      discussion started with a claim that no known single solution
+      (such as reputation or upfront fees) can satisfactorily address
+      the problem by itself without producing too much downside.
```
```suggestion
      the problem by itself without unacceptable downside.
```

LarryRuane commented on _posts/en/newsletters/2023-07-26-newsletter.md:
```diff
@@ -0,0 +1,203 @@
+      supported seeing the proposal tested.
+
+    - *Simplified commitments:* participants discussed the simplified
+      commitments protocol (see [Newsletter #120][news120 commitments]),
+      which is an idea to define which peer is responsible for proposing
```
```suggestion
      commitments protocol (see [Newsletter #120][news120 commitments])
      that defines which peer is responsible for proposing
```

LarryRuane commented on _posts/en/newsletters/2023-07-26-newsletter.md:
```diff
@@ -0,0 +1,203 @@
+      one of the peers didn't want to accept the other's peer's
+      proposal, which is a situation that's complicated to address in
+      the current protocol.  A downside of the simplified commitment
+      approach was that it may increase latency in some cases, as the
```
```suggestion
      approach is that it may increase latency in some cases, as the
```

LarryRuane commented on _topics/en/redundant-overpayments.md:
```diff
@@ -0,0 +1,98 @@
+first try due to forwarding failures, enough of the other parts may
+arrive to allow the receiver to claim their invoiced amount.  Protocol
+features prevent the receiver from claiming more than the invoiced
+amount.  Compared to sending an exact amount, over paying initially can
```
```suggestion
amount.  Compared to sending an exact amount, overpaying initially can
```

harding replied:
@LarryRuane   I'm curious why this instance of single-word "overpaying" is suggested for separation into two words when the rest of the instances in the file are not mentioned?

LarryRuane commented on _topics/en/redundant-overpayments.md:
```diff
@@ -0,0 +1,98 @@
+eliminate the latency of resending failed payments in most cases.
+
+An overpayment requires a mechanism that prevents the receiver from
+claiming more than the invoiced amount.  The simplest mechanisms for
+implementing that is for the spender to not provider the receiver with
```
```suggestion
claiming more than the invoiced amount.  The simplest mechanism for
implementing that is for the spender to not provide the receiver with
```

LarryRuane commented on _topics/en/redundant-overpayments.md:
```diff
@@ -0,0 +1,98 @@
+spender, which may be roughly equivalent to the time needed to handle
+one round of payment failures.  By comparison, in the best case, an
+exact payment will complete on the first try.  That means this type of
+redundant overpayments can (with high probability) put a low ceiling on
```
```suggestion
redundant overpayment can, with high probability, put a low ceiling on
```

LarryRuane commented on _topics/en/redundant-overpayments.md:
```diff
@@ -0,0 +1,98 @@
+
+Other mechanisms may be able to use cryptography to eliminate the
+roundtrip communication overhead, allowing overpayments to complete in
+the same best case amount of time as non-overpayments.  However, that
```
```suggestion
the same best-case amount of time as non-overpayments.  However, that
```

LarryRuane commented on _topics/en/mempool-clustering.md:
```diff
@@ -0,0 +1,70 @@
+mempool for miners in the near term, that criteria can be used instead
+of other heuristics for determining when a transaction [package][topic
+package relay] or [replacement][topic rbf] should be accepted.  It is
+hoped that this can allow transaction relay to provide more flexible
```
```suggestion
hoped that this allows transaction relay to provide more flexible
```

LarryRuane commented on _topics/en/mempool-clustering.md:
```diff
@@ -0,0 +1,70 @@
+## Should be less than 500 characters
+excerpt: >
+  **Mempool clustering** is a proposal to associate each unconfirmed
+  transaction in a mempool with any related transactions, creating a
```
```suggestion
  transaction in a mempool with all related transactions, creating a
```

LarryRuane commented on _topics/en/channel-announcements.md:
```diff
@@ -0,0 +1,64 @@
+  available to forward payments.  The advertisements are relayed through
+  the LN gossip network.
+---
+Version 1 announcements require the nodes prove they control a UTXO for
```
```suggestion
Version 1 announcements require the nodes to prove they control a UTXO for
```
(or "... require that the nodes prove ...")

LarryRuane commented on _posts/en/newsletters/2023-07-26-newsletter.md:
```diff
@@ -167,12 +167,42 @@ answers posted since our last update.*
+- [How can I manually (on paper) calculate a Bitcoin public key from a private key?]({{bse}}118933)
+  Andrew Poelstra gives an overview of computation-by-hand verification
+  techniques, like [codex32][news239 codex32], before walking through how a
+  pubic key could be hand-derived from a private key, a process he estimates would
```
```suggestion
  public key could be hand-derived from a private key, a process he estimates would
```

LarryRuane commented on _posts/en/newsletters/2023-07-26-newsletter.md:
```diff
@@ -167,12 +167,42 @@ answers posted since our last update.*
+  [BIP125][] replaceability.
+
+- [How do route hints affect pathfinding?]({{bse}}118755)
+  Christian Decker explains two reasons for why an LN recipient would provide a sender
```
```suggestion
  Christian Decker explains two reasons an LN recipient would provide a sender
```
Or "... reasons why an ..." (delete "for").
But to me, "reasons" and "why" next to each other seems redundant.

murchandamus commented on _posts/en/newsletters/2023-07-26-newsletter.md:
```diff
@@ -0,0 +1,240 @@
+  take at least 1500 hours, even with process optimizations.
+
+- [Why are there 17 native segwit versions?]({{bse}}118974)
+  Murch explains that [segwit][topic segwit] defined 17 values (0-16) for the [version bytes][bip341
+  witness program] field because of the existing availability of constant
```
I think this should link to BIP141 rather. I would suggest either “witness version” or “version byte field”, but not “version bytes”:

```suggestion
  Murch explains that [segwit][topic segwit] defined 17 values (0-16) for the [witness version][bip141]
  because of the existing availability of constant
```

or: 

```suggestion
  Murch explains that [segwit][topic segwit] defined 17 values (0-16) for the [version byte][bip141]
  field because of the existing availability of constant
```

bitschmidty replied:
Renamed the reference link. Changed 'version bytes' to 'witness version'

murchandamus commented on _topics/en/mempool-clustering.md:
```diff
@@ -0,0 +1,70 @@
+---
+title: Mempool clustering
```
At least from what I’ve been hearing the byword I’ve been hearing for this is:

```suggestion
title: Cluster Mempool
```

bitschmidty commented on _posts/en/newsletters/2023-07-26-newsletter.md:
```diff
@@ -0,0 +1,240 @@
+
+- [Bitcoin Core GUI #740][] updates the PSBT operations dialog to mark outputs
+  paying your own wallet with “own address”. This eases assessment of an
+  imported PSBT’s outcome, especially that the transaction returns change to
```
```suggestion
  imported [PSBT’s][topic psbt] outcome, especially that the transaction returns change to
```

harding commented:
Made edits for all feedback, thanks @LarryRuane @murchandamus @bitschmidty!  Reviewed contributions, thanks @bitschmidty @murchandamus!  Added lede, updated releases/RCs, and added topic entries.

Thanks everyone!

azuchi commented on _posts/en/newsletters/2023-07-26-newsletter.md:
```diff
@@ -0,0 +1,252 @@
+  for DIY Jade devices and a binary for running the main `hwi` program
+  on Apple Silicon hardware with MacOS 12.0+.
+
+- [LDK 0.0.116][] is a relase of this library for creating LN-enabled
```
```suggestion
- [LDK 0.0.116][] is a release of this library for creating LN-enabled
```
